docs/DeductionsApi.md

```diff
@@ -117,9 +117,9 @@ require 'merge_hris_client'
 # setup authorization
 MergeHRISClient.configure do |config|
   # Configure API key authorization: tokenAuth
-  config.api_key['Authorization'] = 'YOUR API KEY'
+  config.api_key['tokenAuth'] = 'YOUR API KEY'
   # Uncomment the following line to set a prefix for the API key, e.g. 'Bearer' (defaults to nil)
-  config.api_key_prefix['Authorization'] = 'Bearer'
+  # config.api_key_prefix['tokenAuth'] = 'Bearer'
```

Comment:
we probably need to figure out a way of doing this, but it would be nice if the Bearer lines in all the docs weren't commented out by default

Reply:
oh i think i know how to do this, will make an asana for q2